reference/src/components/popup/intro/intro.jsx

```diff
@@ -39,23 +39,22 @@ export default class Intro extends Component {
 							We use your data to help bring you personalised content, relevant ads, social media features, and to better understand how you use our website. To do this, we sometimes share this data with social media, advertising, and analytics partners, who may in turn combine it with other data you've given them. Visit our Privacy Policy for more information on our data collection practices.
 						</LocalLabel>
 					</div>
-					<div class={style.acceptBtnWrap}>
+					<div class={style.actions}>
 						<Button
 							class={style.acceptAll}
 							onClick={onAcceptAll}
 						>
 							<LocalLabel localizeKey='acceptAll'>Accept</LocalLabel>
 						</Button>
+						<Button
```

Comment:
I'm also adding an extra button here to "Manage Choices" (also invert=true) that will open the vendor list. Shouldn't we keep the Setting in the same place but style it as a button? @RichardCR-10, any comments on that.
Otherwise we put all three buttons to the right.

Reply:
I propose to move Manage choices under the text and make it as a normal link. Something to discuss

Reply:
Ok, I'll add it as a link underneath then. I was inspired by this popup mentioned in the ticket:
![image](https://user-images.githubusercontent.com/3718400/63759306-8b1fc500-c8bd-11e9-8ebd-e55b7d0069f6.png)